### PR TITLE
#131356227 Redesign Authentication UI

### PR DESCRIPTION
**Pull request by andela-aatanda**
### What does this PR do?
Redesigns Authentication UI i.e SIgnIn/SignUp views

### Description of task to be completed
New SignIn and SignUp view that improves the old view

### Relevant Pivotal tracker story id?
#131356227

### Screenshots
Sign In modal
Sign Up route

### CODE REVIEW

**Comment by andela-foladeji** [comment]
After a failed attempt at logging in, I saw this page
image
You'll probably want to look into that

File: app/views/includes/head.jade [hunk 1]
```diff
@@ -34,6 +34,9 @@ head
 
   link(rel='stylesheet', href='https://fonts.googleapis.com/icon?family=Material+Icons')
 
+  //font awesome CDN
+  script(src="https://use.fontawesome.com/3c82830ea0.js")
```
Comment (andela-foladeji) [inline app/views/includes/head.jade]: What's this for? Is this not enough https://fonts.googleapis.com/icon?family=Material+Icons

File: public/css/style.css [hunk 2]
```diff
@@ -84,6 +84,89 @@
...
+#fb, #twit, #google, #git {
+  width:38%;
+  padding: 0 0.7rem !important;
+  margin-left: 5%;
+  margin-bottom: 15%;
+  margin-top: 5%;
+}
+
+#fb {
+  background-color: #3b5998 !important;
```
Comment (andela-foladeji) [inline public/css/style.css]: Try to minimize (if possible eliminate) your use of !important.
https://developer.mozilla.org/en/docs/Web/CSS/Specificity
selection_012

**Comment by andela-cnwankwo** [comment]
Looks awesome! especially the color combination on the sign in page. However, you could improve on the page responsiveness, a little zoom in or out completely rearranges the icons.
image

File: public/views/index.html [hunk 3]
```diff
@@ -11,15 +11,15 @@
         <li><a ng-click="goTo('how-to-play')">HOW TO PLAY</a></li>
         <li><a ng-click="goTo('charity')">CHARITY</a></li>
         <li ng-show="showOptions"><a href="#!signup">SIGN UP</a></li>
-        <li ng-show="showOptions"><a href="#!signin">SIGN IN</a></li>
         <li ng-hide="showOptions"><a href="/signout">SIGN OUT</a></li>
+        <li ng-show="showOptions"><button class="waves-effect waves-light btn" data-target='modal1' id='signin'>SIGN IN</button></li>
```
Comment (andela-foladeji) [inline public/views/index.html]: For consistency with other nav links use this
`<li ng-show="showOptions" data-target='modal1'><a>SIGN IN</a></li>`

File: public/css/style.css (hunk 2 above)
Comment (azemoh) [inline public/css/style.css]: Also stick to using class selectors in your CSS.

File: app/views/includes/head.jade (hunk 1 above)
Comment (andela-aatanda) [inline app/views/includes/head.jade]: Thought of that too but Material Design does not have social icons.

File: public/views/signup.html [hunk 4]
```diff
@@ -61,7 +61,7 @@ <h5 class="center-align">Or create a username/password and select an avatar:</h5
             <div id="contain-avatars">
               <ul ng-controller="IndexController" class="avatar-list">
                 <li ng-repeat="avatar in avatars" class="avatars ng-scope">
-                  <input name="avatar" type='radio' value="{{$index}}">
+                  <input name="avatar" type='checkbox' value="{{$index}}" class="avatar">
```
Comment (azemoh) [inline public/views/signup.html]: You should use radio button so a user can only select one avatar.

File: public/css/style.css [hunk 5]
```diff
@@ -84,6 +84,108 @@
...
+
+#social a:hover {
+  color: black !important;
+}
+
+form {
+  margin-left: 0px !important;
+}
+
+.btn{
```
Comment (neddinn) [inline public/css/style.css]: Maintain consistency. Add a space after properties.

**Pull request merged**